Project/core/src/main/java/View/screen/GameScreenComponents/AskButton.java

```diff
@@ -47,6 +52,11 @@ public Actor hit(float x, float y, boolean touchable) {
 
     @Override
     public void draw(Batch batch, float parentAlpha) {
+        this.player = gameController.getGame().getCurrentPlayer();
+        if (player.getClass().equals(RougeAgent.class)){
+            super.draw(batch, parentAlpha); // Important
```

Comment:
Same issue as before, buttons invisible but active. need to disable here i think?

Reply:
Yes, we can add it to issue #40 I suggest.